### PR TITLE
Remove inline source/parser names 

### PR DESCRIPTION
**Pull request by MrAnno**

### CODE REVIEW

**Comment by fekete-robert** [comment]
Thanks a lot!

File: en/common/chunk/example-junctions-syslog-parser.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <title>Using junctions</title>
     <para>For example, suppose that you have a single network source that receives log messages from different devices, and some devices send messages that are not RFC-compliant (some routers are notorious for that). To solve this problem in earlier versions of &abbrev;, you had to create two different network sources using different IP addresses or ports: one that received the RFC-compliant messages, and one that received the improperly formatted messages (for example, using the <userinput>flags(no-parse)</userinput> option). Using junctions this becomes much more simple: you can use a single network source to receive every message, then use a junction and two channels. The first channel processes the RFC-compliant messages, the second everything else. At the end, every message is stored in a single file. The filters used in the example can be <parameter>host()</parameter> filters (if you have a list of the IP addresses of the devices sending non-compliant messages), but that depends on your environment.</para>
     <synopsis>log {
-    source s_network { syslog(ip(10.1.2.3) transport("tcp") flags(no-parse)); };
```
Comment (faxm0dem) [inline en/common/chunk/example-junctions-syslog-parser.xml]: wasn't the `ip()` syntax removed?

Comment (fekete-robert) [inline en/common/chunk/example-junctions-syslog-parser.xml]: Hi,
Not that I know of. AFAIK it is required in the source, and not needed in
the destination.

**Comment by faxm0dem** [comment]
you're right, sorry for the confusion

**Pull request merged**